### ISSUE TITLE
lscpu: failed to determine number of CPUs: /sys/devices/system/cpu/possible: No such file or directory

### ISSUE THREAD

**Issue by cneira** [joyent/illumos-joyent#140]
on a lxzone trying to use lscpu results in the following error:
```bash
lscpu: failed to determine number of CPUs: /sys/devices/system/cpu/possible: No such file or directory
```
I could get the information using /proc/cpuinfo but I the original script used lscpu for this

**Comment by jjelinek** [comment]
Thanks, I filed OS-6019 for this.

**Comment by jjelinek** [comment]
This specific issue was fixed with commit bdeddd1 but I am leaving it open for now since lscpu still doesn't work. Now with this message: cannot open /sys/devices/system/node/node0/cpumap.

Now that we have re-enabled this portion of the sysfs device tree, we can finish the changes to enable lscpu. This is being tracked as OS-6044.

**Comment by jjelinek** [comment]
I have added cpumap with 920ae14, and lscpu is working now.
